bundles/org.openhab.binding.icalendar/README.md

```diff
@@ -0,0 +1,99 @@
+| `username`          | The username for pulling the calendar. If set, the binding pulls the calendar using basic auth. Only valid in combination with `password`                                                                                                                                  | optional  |
+| `password`          | The password for pulling the calendar. If set, the binding pulls the calendar using basic auth. Only valid in combination with `username`                                                                                                                                  | optional  |
+| `authorizationCode` | The authorization code to permit the execution of embedded Command Tags. If set, the binding checks that the authorization code in the Command Tag matches before executing any commands.                                                                                  | optional  |
+
+## Channels
+
+The channels describe the current and the next event. They are all read-only.
+
+| Channel           | Type      | Description                         |
+|-------------------|-----------|-------------------------------------|
+| current_presence  | Switch    | Current presence of a event         |
+| current_title     | String    | Title of a currently present event  |
+| current_start     | DateTime  | Start of a currently present event  |
+| current_end       | DateTime  | End of a currently present event    |
+| next_title        | String    | Title of the next event             |
+| next_start        | DateTime  | Start of the next event             |
+| next_end          | DateTime  | End of the next event               |
+
+## Command Tags
+
+Each calendar event may include one or more Command Tags in its description text. These Command Tags are used to issue commands directly to other Items in the system when the event begins or ends. A Command Tag must consist of at least three fields. A fourth field is optional. The syntax is as follows:
+
+	BEGIN:Item_Name:New_State_Value
+	BEGIN:Item_Name:New_State_Value:Authorization_Code
+	END:Item_Name:New_State_Value
+	END:Item_Name:New_State_Value:Authorization_Code
+
+The first field **must** be either `BEGIN` or `END`. If it is `BEGIN` then the command will be executed at the beginning of the calendar event. If it is `END` then the command will be executed at the end of the calendar event. A calendar event may contain multiple `BEGIN` or `END` tags. If an event contains both `BEGIN` and `END` tags, this Items (say) to be turned ON at the beginning of an event and turned OFF again at the end of the event.
+ 
+The `Item_Name` field must be the name of any Item in the system.
+
+The `New_State_Value` is the state value that will be sent to the Item. It must be a value which is compatible with the Item type. See OpenHAB core definitions for [command types](https://www.openhab.org/docs/concepts/items.html#state-and-command-type-formatting) for valid types and formats.
```

Comment:
"openHAB"

Reply:
OK

Reply:
This hasn't been addressed.

Reply:
> For some reason, this edit wasn't showing up, but now I see it. Weird.

Yeah. GitHub currently seems to be very slow at synchronizing